### DISCUSSION TITLE
Hosting a Next + Keystatic project on Github pages.

### DISCUSSION THREAD

**Discussion by Hamzah-Ahmad** [Thinkmill/keystatic#1388] [Q&A]
I have created a simple Next.js and Keystatic project that is working correctly on local. I would like to deploy the project on Github pages.
That involves updating the output mode to `export` in the `next.config.js` file. However, this step leads to the following error:

```
Error: Page "/keystatic/[[...params]]" is missing "generateStaticParams()" so it cannot be used with "output: export" config.
```

Are there any workarounds for this? I am only interested in accessing the admin UI locally, if that helps.

Thank you

**Comment by tordans** [comment]
I just documented the setup we use at https://github.com/Thinkmill/keystatic/discussions/1361. That should help you understand why this does not work the way you expect it and how to set it up  for you Next + Keystatic Site.
